src/lib/services/types/wasm/contract.ts

```diff
@@ -41,26 +35,6 @@ const zContractRest = z.object({
 });
 export type ContractRest = z.infer<typeof zContractRest>;
```

Comment:
ContractLcd?

Reply:
@songwongtp does it already included in your revamp?

Reply:
duplicated, already have a type named ContractLcd